hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -0,0 +1,285 @@
+                            <div key={pendingOrder.id}>
+                                <GeneralOrderTableTitle
+                                    orderId={pendingOrder.id}
+                                    orderStatus={pendingOrder.status}
+                                />
+                                <TableContainer
+                                    component={Paper}
+                                    elevation={2}
+                                    square={true}
+                                >
+                                    <Table className={styles.table} size="small">
+                                        <TableHead>
+                                            <TableRow>
+                                                <TableCell
+                                                    className={styles.widthFixed}
+                                                />
+                                                <TableCell className={styles.width6}>
+                                                    Name
+                                                </TableCell>
+                                                <TableCell
+                                                    className={`${styles.width1} ${styles.noWrap}`}
+                                                >
+                                                    Model
+                                                </TableCell>
+                                                <TableCell
+                                                    className={`${styles.width1} ${styles.noWrap}`}
+                                                >
+                                                    Manufacturer
+                                                </TableCell>
+                                                <TableCell
+                                                    className={`${styles.width1} ${styles.noWrap}`}
+                                                >
+                                                    Qty requested
+                                                </TableCell>
+                                                <TableCell
+                                                    className={`${styles.width6} ${styles.noWrap}`}
+                                                >
+                                                    Qty granted
+                                                </TableCell>
+                                                <TableCell
+                                                    className={`${styles.width1} ${styles.noWrap}`}
+                                                >
+                                                    Time
+                                                </TableCell>
+                                                <TableCell>
+                                                    <Checkbox
+                                                        color="primary"
```

Comment:
remove the margin here, instead can you do `<TableCell align="center">`, that will center your checkbox